paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/samples/ui/wallet/SavedPaymentMethodsActivity.kt

```
@@ -47,7 +62,19 @@ internal class SavedPaymentMethodsActivity : AppCompatActivity() {
 
                     when (val state = viewState) {
                         is SavedPaymentMethodsViewState.Data -> {
-                            PaymentDefaults()
+                            PaymentDefaults(
+                                onUpdateDefaultPaymentMethod = {
+                                    try {
+                                        savedPaymentMethodsSheet.present()
+                                    } catch (_: Error) {
```

Should we just let it crash for now?

---

I like this for now!